roles/identity-management/manage-idm-identities/README.md

```diff
@@ -13,6 +13,7 @@ Role Variables
 |**ipa_admin_user**|The IPA/IdM admin user with proper permissions to administer identities|yes|N/A|
 |**ipa_admin_password**|The IPA/IdM admin password for the above mentioned admin user|yes|N/A|
 |**ipa_validate_certs**|Whether to validate the IPA/IdM certificate|no|True|
+|**ipa_password_reset_users**|A list of users who's passwords you wish to reset|no|[]|
```

Comment:
I'd rather that we set a flag in the `identity` dictionary for the user(s) that we'd like to reset the password for. This will make the use of the `set-user-flags.py` easier ... i.e.: just a flag as part of the `entry` variable (could be as simple as just removing the `has_password` key)

Reply:
Would this mean including the password reset flag in the csv file for the `process_and_notify_users` playbook? If so, doesn't that create risk accidentally resetting the wrong users passwords? For example if you set the flag to true and later ran the playbook again and forgot to set it back to false.